Makefile

```diff
@@ -78,12 +78,16 @@ check: fmt vet lint test ## Check your code
 
 .PHONY: unit
 unit: # Run unit test
-	$(DOCKER_CMD) go test -race -cover ./cmd/... ./pkg/...
+	$(DOCKER_CMD) go test -v -race -cover ./cmd/... ./pkg/...
 
 .PHONY: integration
 integration: ## Run integration test
 	$(DOCKER_CMD) go test -v sigs.k8s.io/cluster-api-provider-aws/test/integration
 
+.PHONY: build-e2e
+build-e2e:
+	go test -c -o bin/e2e.test sigs.k8s.io/cluster-api-provider-aws/test/machines
```

Comment:
Was wondering about the `-c` here; when/where is the binary run?

```
	-c
	    Compile the test binary to pkg.test but do not run it
	    (where pkg is the last element of the package's import path).
	    The file name can be changed with the -o flag.
```

Reply:
Inside the CI. Underlying Jenkins job runs `make build-e2e` and expects the binary under `bin/e2e.test`.

Reply:
Would be useful to add your answer as a comment for the next person. 👍